PicPi/FullImageViewController.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  FullImageViewController.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 1/2/21.
+//
+
+import UIKit
+import Promises
+
+class FullImageViewController: UIViewController {
+  private let deleteButton = customBtnRoundCornerBlueWithShadow(type: .custom)
+  private var margins: UILayoutGuide!
+  private var photo: Photo
+  
+  init(photo: Photo) {
+    self.photo = photo
+    super.init(nibName: nil, bundle: nil)
+  }
+  
+  @available(*, unavailable)
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    margins = view.layoutMarginsGuide
+    view.backgroundColor = .white
+    title = photo.filename
+    
+    setupDeleteButton()
+    setupImageView()
+  }
+  
+  private func setupImageView() {
+    let imageView = UIImageView()
+    imageView.isUserInteractionEnabled = true
+    imageView.translatesAutoresizingMaskIntoConstraints = false
+    imageView.contentMode = .scaleAspectFit
+    imageView.frame = CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height)
+    /// download full image from Pi
+    imageView.kf.setImage(with: photo.fullImage)
+    view.addSubview(imageView)
+    
+    NSLayoutConstraint.activate(
+      [
+        imageView.leadingAnchor.constraint(equalTo: margins.leadingAnchor),
+        imageView.trailingAnchor.constraint(equalTo: margins.trailingAnchor),
+        imageView.topAnchor.constraint(equalTo: margins.topAnchor ),
+        imageView.bottomAnchor.constraint( equalTo: deleteButton.topAnchor ),
+      ]
+    )
+  }
+  
+  private func setupDeleteButton() {
+    view.addSubview(deleteButton)
+    ///add action when search button pressed
```

Comment:
Nit: update comment or remove if not relevant

Reply:
Done!